x-pack/plugins/apm/public/components/app/service_logs/index.tsx

```diff
@@ -99,7 +99,7 @@ export const getInfrastructureKQLFilter = (
 
   const kqlFilter = containerIds.length
     ? containerIds.map((id) => `${CONTAINER_ID}: "${id}"`)
-    : hostNames.map((id) => `${HOSTNAME}: "${id}"`);
+    : hostNames.map((id) => `${HOST_NAME}: "${id}"`);
```

Comment:
does this work for service logs? should we fallback to host.hostname here as well?

Reply:
I don't know, probably yes. Maybe in a different issue? Does this relate to #120694?

Reply:
This should be fine. Logs/metrics app also used `host.name`